src/serial.rs

```diff
@@ -72,6 +72,14 @@ impl SerialError {
     }
 }
 
+impl fmt::Display for SerialError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.err)
```

Comment:
`std::io::ErrorKind` implements `std::fmt::Display` since Rust 1.60.
I don't have a strong opinion if I should either set the MSRV to 1.60 or just display the debug format output of `std::io::ErrorKind`.
Let me know and I'll make the change.